Example/Swift Example/View Controllers/UserAuthenticationViewController.swift

```diff
@@ -204,6 +214,61 @@ class UserAuthenticationViewController: UITableViewController {
         }
     }
 
+    func requestVerification() {
+        guard let user = SKYContainer.default().auth.currentUser else {
+            return
+        }
+
+        let alert = UIAlertController(title: "Request Verification", message: "Select the field for verification. This example only supports email and phone.", preferredStyle: .alert)
+
+        let verifyableRecordKeys = ["email", "phone"]
+        for recordKey in verifyableRecordKeys {
+            let enabled = (user[recordKey] as? String)?.isEmpty == false
+            let title = enabled ? recordKey : "\(recordKey) (missing)"
+            let action = UIAlertAction(title: title, style: .default, handler: { (_) in
+                SKYContainer.default().auth.requestVerification(recordKey, completion: { (error) in
+                    if let error = error {
+                        self.showError(error, completion: nil)
+                        return
+                    }
+                })
```

Comment:
For request verification success, better to show alert with message request verification success? Now there are no  ui indicator to show that the request is success.